Mage/Command/BuiltIn/CompileCommand.php

```diff
@@ -22,19 +22,43 @@
 class CompileCommand extends AbstractCommand
 {
     /**
+     * @var Compiler
+     */
+    private $compiler;
+
+    public function __construct()
+    {
+        $this->setCompiler(new Compiler());
+    }
+
+    public function setCompiler(Compiler $compiler)
```

Comment:
If you want to keep the setter and the possibility to set the compiler on runtime, the doc-block is missing for this method.